### PR TITLE
[BXMSPROD-851] - Upgrade jdk for 8 and 11 on openstack nodes to the latest versions

### PR DESCRIPTION
**Pull request by akoufoudakis**
**Thank you for submitting this pull request**

**JIRA**: _https://issues.redhat.com/browse/BXMSPROD-851_

### CODE REVIEW

File: jenkins-slaves/ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -20,7 +20,8 @@
     pkg: ['wget', 'mc', 'vim', 'unzip', 'iputils', 'bind-utils', 'findutils',
           'htop', 'screen', 'lsof', 'bash-completion', 'bzip2', 'rng-tools',
           'ansible', 'zip', 'python2-pip', 'jq', 'libXScrnSaver', 'python3', 'python3-devel',
-          'python3-pip', 'glibc-devel', 'zlib-devel', 'podman', 'gcc', 'krb5-devel', 'libstdc++-static']
+          'python3-pip', 'glibc-devel', 'zlib-devel', 'podman', 'gcc', 'krb5-devel', 'libstdc++-static', 
+          'java-1.8.0-openjdk-devel.x86_64', 'java-11-openjdk-devel.x86_64']
```
Comment (winklerm) [inline jenkins-slaves/ansible/roles/common/tasks/main.yml:24]: Do we need .x86_64 suffix here?

**Review by winklerm — CHANGES_REQUESTED** [review]
Looks good to me except the x86_64 suffix for dnf packages - I think it is redundant; however, it does not hurt.

**Review by winklerm — APPROVED** [review]
Thanks, looks good to me!

**Review by mareknovotny — APPROVED** [review]
when merged please handle properly the switching to the new image too.

**Pull request merged**